<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" ignored="0" total="7" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="247787" started-at="2022-01-28T14:40:24Z" finished-at="2022-01-28T14:44:32Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="247787" started-at="2022-01-28T14:40:24Z" finished-at="2022-01-28T14:44:32Z">
      <class name="com.akBanking.testCases.TC_AddCustomertest_003">
        <test-method status="PASS" signature="setup(java.lang.String)[pri:0, instance:com.akBanking.testCases.TC_AddCustomertest_003@71c8becc]" name="setup" is-config="true" duration-ms="12135" started-at="2022-01-28T14:43:57Z" finished-at="2022-01-28T14:44:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="addNewCustomer()[pri:0, instance:com.akBanking.testCases.TC_AddCustomertest_003@71c8becc]" name="addNewCustomer" duration-ms="22504" started-at="2022-01-28T14:44:09Z" finished-at="2022-01-28T14:44:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNewCustomer -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.akBanking.testCases.TC_AddCustomertest_003@71c8becc]" name="teardown" is-config="true" duration-ms="702" started-at="2022-01-28T14:44:31Z" finished-at="2022-01-28T14:44:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.akBanking.testCases.TC_AddCustomertest_003 -->
      <class name="com.akBanking.testCases.TC_Logintest_001">
        <test-method status="PASS" signature="setup(java.lang.String)[pri:0, instance:com.akBanking.testCases.TC_Logintest_001@3745e5c6]" name="setup" is-config="true" duration-ms="152946" started-at="2022-01-28T14:40:26Z" finished-at="2022-01-28T14:42:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="loginTest()[pri:0, instance:com.akBanking.testCases.TC_Logintest_001@3745e5c6]" name="loginTest" duration-ms="3985" started-at="2022-01-28T14:42:59Z" finished-at="2022-01-28T14:43:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.akBanking.testCases.TC_Logintest_001@3745e5c6]" name="teardown" is-config="true" duration-ms="682" started-at="2022-01-28T14:43:03Z" finished-at="2022-01-28T14:43:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.akBanking.testCases.TC_Logintest_001 -->
      <class name="com.akBanking.testCases.TC_LogintestDP_002">
        <test-method status="PASS" signature="setup(java.lang.String)[pri:0, instance:com.akBanking.testCases.TC_LogintestDP_002@5e4c8041]" name="setup" is-config="true" duration-ms="5761" started-at="2022-01-28T14:43:03Z" finished-at="2022-01-28T14:43:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="loginDDT(java.lang.String, java.lang.String)[pri:0, instance:com.akBanking.testCases.TC_LogintestDP_002@5e4c8041]" name="loginDDT" duration-ms="11958" started-at="2022-01-28T14:43:12Z" data-provider="LoginData" finished-at="2022-01-28T14:43:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr377698]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[vUbyzAb]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method status="PASS" signature="loginDDT(java.lang.String, java.lang.String)[pri:0, instance:com.akBanking.testCases.TC_LogintestDP_002@5e4c8041]" name="loginDDT" duration-ms="12124" started-at="2022-01-28T14:43:24Z" data-provider="LoginData" finished-at="2022-01-28T14:43:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr377698]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[vUbyzAb]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method status="FAIL" signature="loginDDT(java.lang.String, java.lang.String)[pri:0, instance:com.akBanking.testCases.TC_LogintestDP_002@5e4c8041]" name="loginDDT" duration-ms="6300" started-at="2022-01-28T14:43:36Z" data-provider="LoginData" finished-at="2022-01-28T14:43:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr137319]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[bAdYvyb]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.akBanking.testCases.TC_LogintestDP_002.loginDDT(TC_LogintestDP_002.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method status="FAIL" signature="loginDDT(java.lang.String, java.lang.String)[pri:0, instance:com.akBanking.testCases.TC_LogintestDP_002@5e4c8041]" name="loginDDT" duration-ms="6258" started-at="2022-01-28T14:43:42Z" data-provider="LoginData" finished-at="2022-01-28T14:43:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr164225]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[jahetAp]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.akBanking.testCases.TC_LogintestDP_002.loginDDT(TC_LogintestDP_002.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method status="FAIL" signature="loginDDT(java.lang.String, java.lang.String)[pri:0, instance:com.akBanking.testCases.TC_LogintestDP_002@5e4c8041]" name="loginDDT" duration-ms="5603" started-at="2022-01-28T14:43:49Z" data-provider="LoginData" finished-at="2022-01-28T14:43:54Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr112909]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[tytUreg]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.akBanking.testCases.TC_LogintestDP_002.loginDDT(TC_LogintestDP_002.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.akBanking.testCases.TC_LogintestDP_002@5e4c8041]" name="teardown" is-config="true" duration-ms="2439" started-at="2022-01-28T14:43:54Z" finished-at="2022-01-28T14:43:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.akBanking.testCases.TC_LogintestDP_002 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
